diagnostic.md

```diff
@@ -44,8 +56,21 @@ How would we reference each of the following categories of elements?
 -   Only those elements belonging to both the `important` and `red` _classes_
 -   All `p` elements whose parent elements are `div`s
 
-<!-- your answer starts here -->
+.big{
+  style: value;
+}
+
+#contentPane {
+  style: value;
+}
+
+.important, .red {
+  style: value;
+}
 
+div p {
```

Comment:
This is close. It selects all <p> elements that are inside <div> elements (at any level).
If I want only the <p>'s whose *direct parent* is a <div>, I need `div > p`.